org.eclipse.xtext.ide/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-ActivationPolicy: lazy
 Bundle-Vendor: Eclipse Xtext
 Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.xtend.lib,
- org.eclipse.core.runtime;bundle-version="3.6.0",
+ org.eclipse.core.runtime;bundle-version="3.6.0";resolution:=optional;x-installation:=greedy,
+ org.eclipse.equinox.common;bundle-version="3.5.0",
```

Comment:
why 3.5 here while core.runtime would be 3.6?

Reply:
copy&paste from xtext-core?!?

Reply:
OK, if this valid for o.e.x then it should be fine.